pynos/versions/ver_7/ver_7_1_0/interface.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 Great Software Laboratory Pvt. Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from pynos.versions.ver_7.ver_7_0_0.yang.brocade_interface \
```

Comment:
We have moved the port_profile_port function to base.interface.py.
Also removed the corrosponding import entry form device.py